media-libs/noise-suppression-for-voice/noise-suppression-for-voice-1.03.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A real-time noise suppression plugin for voice"
+HOMEPAGE="https://github.com/werman/noise-suppression-for-voice"
+
+if [ "${PV}" = "9999" ]; then
```

Comment:
Bash tests where we can, as we're in Bash: https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101.